ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -55,26 +93,43 @@ export default function EditGasPopover({
     if (!transaction || !mode) {
       closePopover();
     }
+
+    const cancelSpeedUpGas =
+      gasEstimateType === GAS_ESTIMATE_TYPES.FEE_MARKET
+        ? {
+            gasLimit: decGWEIToHexWEI(gasLimit),
```

Comment:
gasLimit is not in gwei, it needs to be just a straight conversion to hex. decimalToHex.